### PR TITLE
Wareframe

### PR DESCRIPTION
**Pull request by Jeanbulambo**
In this PR, we implemented the following :
- We delete all the boilerplate from CRA (text, images, styles).
- We created a directory called components and Inside components and a new Calculator.js file.
- In Calculator.js, create a React component that matches this design:

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import '../Style.css';
+
+export default class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
```
Comment (ShoiraTa) [inline src/components/Calculator.js:8]: - [ ] Good job implementing this component as class based 👍🏻

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Calculator from './components/Calculator';
+
+function App() {
+  return (
+    <Calculator />
+  );
+}
+
```
Comment (ShoiraTa) [inline src/App.js]: - [ ] Kindly rewrite `App.js` to class based component as well 😉 
- [ ] Something looks to be broken on UI side, kindly check it:
Screenshot 2022-08-10 at 10 59 25

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @Jeanbulambo ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Professional Documentation! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — APPROVED** [review]
# Approved 🎉

Hi @Jeanbulambo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**